src/test/java/com/hyperlocal/server/MerchantControllerTest.java

```diff
@@ -0,0 +1,34 @@
+package com.hyperlocal.server;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.verify;
+
+import com.google.gson.JsonParser;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Spy;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+public class MerchantControllerTest {
+
+  private final String MERCHANT_DATA_AS_STRING = "{\"MerchantID\": \"4\",\"MerchantName\":\"Test Merchant\", \"MerchantPhone\":\"7865466876\"}";
+  @Spy
```

Comment:
It would be better to represent this as a proper Gson and then convert it to string while comparing.

Reply:
Done